src/utils.ts

```diff
@@ -103,7 +103,8 @@ export function jitiInteropDefault(ctx: Context, mod: any) {
 }
 
 function interopDefault(mod: any): any {
```

Comment:
`interopDefault` seems to be an important function. I'd add an extra comment explaining what it is about since the function name isn't self-explanatory. Thanks

Reply:
PR welcome to improve readme explaining it better 👍🏼